.github/workflows/ci.yml

```diff
@@ -1,23 +1,16 @@
-name: ci
+name: release
```

Comment:
What happened in this file? 😄

Reply:
Seems too early fork, missing latest changes 😄 - have done it also a few times.